Formula/gnupg.rb

```diff
@@ -42,7 +42,8 @@ def install
   end
 
   def caveats; <<-EOS.undent
-    All commands have been installed with the suffix '1'.
+    This formula does not install either `gpg` or `gpgv` vanilla executables
+    into the PATH. All executables have been installed with a "1" suffix.
```

Comment:
I might avoid "vanilla" since it's kinda idiomatic. Maybe something like:

> This formula installs tools named gpg1 and gpgv1. If you need to use these tools without the "1" suffix, you may...

Exciting changes afoot!